djangocms_url_manager/forms.py

```diff
@@ -235,15 +258,14 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         # Set url if object exists
-        if self.instance and self.instance.url_id:
-            self.fields["url"].initial = self.instance.url_id
+        if self.instance and self.instance.url_grouper_id:
+            self.fields["url_grouper"].initial = self.instance.url_grouper.url()
```

Comment:
```suggestion
            self.fields["url_grouper"].initial = self.instance.url_grouper_id
```
Tested this change on my local/